wsd/DocumentBroker.cpp

```diff
@@ -1704,15 +1706,26 @@ static std::string extractViewSettings(const std::string& viewSettingsPath,
         auto result = parser.parse(ifs);
 
         const Poco::JSON::Object::Ptr& viewsetting = result.extract<Poco::JSON::Object::Ptr>();
+        std::string accessibilityState, zoteroAPIKey, signatureCertificate, signatureKey, signatureCa;
 
-        std::string accessibilityState, zoteroAPIKey;
         JsonUtil::findJSONValue(viewsetting, "accessibilityState", accessibilityState);
+        _isViewSettingsAccessibilityEnabled = accessibilityState == "true";
```

Comment:
The idea with the accessibility view setting was that if it was present and set then it would override anything else.

What will happen for signature stuff for a user if there were the old settings and then they run a new version of coolwsd, do they lose their old settings. I see there was some migration effort for zotero settings at https://github.com/CollaboraOnline/online/pull/13091 do we need something like that or some conditional thing to not set _isViewSettingsUpdated if the json values are not present?